models/news.models.js

```diff
@@ -48,9 +48,28 @@ function fetchCommentsByArticleId(id) {
   });
 }
 
+function updateArticleVotes(id, { inc_votes }) {
+  if (inc_votes) {
+    const valueArr = [inc_votes, id];
+
+    const sqlQuery = `
+    UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *`;
+
+    return db.query(sqlQuery, valueArr).then(({ rows: [updatedArticle] }) => {
+      return updatedArticle;
+    });
+  } else {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
```

Comment:
I wonder if you have no inc_votes this value would be undefined and your postgres database would throw an error but this is also currently fine